### PR TITLE
fix: record button now has a default position and is draggable

### PR DESCRIPTION
**Pull request by RohanPunjani**
- Record Button is now centered horizontally and positioned 200px above the bottom
- Removed the shadow from the cropper window to prevent unexpected behavior, such as draggable and fullscreen
- `update_crop_area` doesn't take button coords anymore

### CODE REVIEW

File: src-tauri/src/cropper/mod.rs [hunk 1]
```diff
@@ -1,10 +1,18 @@
 use crate::AppState;
-use tauri::{AppHandle, Manager, Position, WebviewUrl, WebviewWindowBuilder};
+use tauri::{AppHandle, LogicalSize, Manager, WebviewUrl, WebviewWindowBuilder};
+use windows::Win32::UI::WindowsAndMessaging::{SetWindowDisplayAffinity, WINDOW_DISPLAY_AFFINITY};
```
Comment (Pranav2612000) [inline src-tauri/src/cropper/mod.rs:3]: Shoud this `use` be moved inside the "windows" cfg? Similar to 'cocoa' for MacOS

**Comment by clearlysid** [comment]
Hmm, I'm able to "break" this feature in some ways. Under certain specific situations, the record button window comes out like a black box -- which is kinda worse than the window itself.

How to reproduce it:
1. Ctrl + shift + 2
2. Make an area
3. Don't start recording, but make another area (that includes the record button win)
4. Now record
5. you will have the black box in your recording


Screenshot 2024-05-10 at 11 04 02 AM

**Comment by clearlysid** [comment]
the last commit is important^ fixes an upstream windows-capture dependency issue.

**Comment by clearlysid** [comment]
Merging this minus the Windows cropper and record window exclusion fix for now, will create a separate issue for it.

**Pull request merged**